Test/git-issues/git-issue-2956b.js

```diff
@@ -0,0 +1 @@
+global.OneOne = () => _dafny.Seq.of(1);
```

Comment:
I would add a comment here, since this is a (purposeful) abuse of the API

Reply:
Good idea, will do.
```suggestion
// The JS compiler would normally convert this integer value to a BigNumber.
// Omitting the conversion tests an edge case that would not arise during normal compilation.
global.OneOne = () => _dafny.Seq.of(1);
```